OHCRN/platform
# 💄#347: Add radio button component

## Description of Changes

Created a reusable React radio button selection component for step 3 & 4 of the consent wizard. Comes with disabled + error states

### To test locally/see the component:
Copy the changes from the commit "5cf7daa: 💄#347: Add test form to see radio button", and then run dev. Then go to [http://localhost:3000/en/invite](http://localhost:3000/en/invite) to see the button. I did this by creating a temporary TestForm component.

## PR Readiness Checklist

- [x] "Expected Outcome(s)" in ticket have been met
- [x] Ticket number included in PR title
- [x] Connected ticket to PR
- [x] Labels added to PR for service name (`consent-api`, `data-mapper`, etc...), type (`chore`, `documentation`, etc...), status (`draft`, `on-hold`, etc...) **if applicable**
- [x] Builds locally without errors or warnings
- [x] Tests are updated (if required) and passing
- [ ] PR feedback has been addressed **if applicable**
- [x] New environment variables added to `.env.schema` files, `README.md`
- [x] Added copyrights to new files
